nymvpn-x/src-tauri/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name = "nymvpn-x"
+version = "0.1.0-dev"
+description = "NymVPN Desktop Client"
+authors = ["pierre <dommerc.pierre@gmail.com>", "zane <zanecschepke@gmail.com>", "nym"]
+license = "GPL-3.0-only"
+repository = ""
+edition = "2021"
+
+[build-dependencies]
+tauri-build = { version = "1.5", features = [] }
+build-info-build = "0.0.34"
+
+[dependencies]
+tauri = { version = "1.6.0", features = [ "window-set-size", "os-all", "updater", "process-all", "shell-open"] }
+tokio = { version = "1.33", features = ["rt", "sync", "time", "fs"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+tracing = "0.1"
+tracing-subscriber =  { version = "0.3.1", features = ["tracing-log", "env-filter"] }
+anyhow = "1.0"
+dotenvy = "0.15.7"
+thiserror = "1.0"
+ts-rs = { version = "7.0.0", features = ["chrono-impl"] }
+once_cell = "1.18.0"
+toml = "0.8.5"
+time = "0.3.9"
+# TODO use the crate once it's published, for now use local path
+nym-vpn-lib = { path = "../../nym-vpn-lib" }
```

Comment:
Don't think you need this now? I think you can use nym-gateway-directory directly and not have to worry about building wireguard and all that stuff

Reply:
to the only usage I see is there https://github.com/nymtech/nym-vpn-client/pull/368/files#diff-176db7941721b48c4835c8d375fb267849129e94b3056cc25dfdf5b2c4d8e37f

![image](https://github.com/nymtech/nym-vpn-client/assets/6359431/2ab54321-50ec-4928-88d0-f54fb729edf5)

so it depends if we still need these calls or not?